server/dataone_package.py

```diff
@@ -101,18 +144,22 @@ def get_file_md5(file_object, md5):
         return None
     handle.close()
     return md5
+    logger.debug('Leaving get_file_md5')
```

Comment:
this logger statement can't get called due to the return directly above it

Reply:
Good catch-just made the change